libpages/root.go

```diff
@@ -49,6 +49,19 @@ func (t RootType) String() string {
 	}
 }
 
+func tlfTypeToPath(t tlf.Type) string {
+	switch t {
+	case tlf.Private:
+		return string(libkbfs.PrivatePathType)
```

Comment:
Technically the "right" variables to use would be the ones in `libgit/autogit_node_wrappers.go", which define how the tlf types map to subdirs for autogit specifically.  (They would have to be exported.)

But this is fine too I guess, just a little dirtier.

Reply:
Ah I only saw the ones in `libkbfs`. I'll export it from `libgit`!